exercises/E-conditionals/exercise.js

```diff
@@ -9,6 +9,12 @@
 let name = "Daniel";
 let danielsRole = "mentor";
 
+if (danielsRole == " student") {
```

Comment:
Neat!
Lines 13 and 15 are identical though, do you think the IF statement is necessary at all in this case? I know that was the requirement, but your solution actually removes the need for the IF statement, so well done